docs/language.md

```diff
@@ -250,7 +250,13 @@ For user space symbols, symbolicate lazily/on-demand (`true`) or symbolicate eve
 
 Default: "GPL"
 
-The license bpftrace will use to load BPF programs into the linux kernel.
+The license bpftrace will use to load BPF programs into the linux kernel. Here is the list of accepted license strings:
```

Comment:
Nit: explain why and link to a the kernel docs?

Reply:
Added a link to the other part of the language doc.